configuration/pih/petl/jobs/hiv/viral_load/source.sql

```diff
@@ -0,0 +1,126 @@
+SET sql_safe_updates = 0;
+
+DROP TABLE IF EXISTS temp_hiv_constructs;
+DROP TABLE IF EXISTS temp_viral_load;
+DROP TABLE IF EXISTS temp_vl_sample_taken_date;
+DROP TABLE IF EXISTS temp_days_since_vl;
+
+-- hiv constructs table
+CREATE TEMPORARY TABLE temp_hiv_constructs
+(
+patient_id 				INT,
+encounter_id 			INT
+);
+
+INSERT INTO temp_hiv_constructs (patient_id, encounter_id)
+SELECT person_id, encounter_id FROM obs WHERE voided = 0 AND concept_id = CONCEPT_FROM_MAPPING("PIH", "HIV viral load construct") AND encounter_id IN (SELECT encounter_id FROM encounter
+WHERE encounter_type =(SELECT encounter_type_id FROM encounter_type WHERE name = "Laboratory Results"));
```

Comment:
Right now I think this is the only encounter type that has VL results, but is there any harm in removing this clause and just searching on the "HIV viral load construct" just in case this changes in the future?

Reply:
Done

Reply:
I agree!